README.md

```diff
@@ -6,8 +6,9 @@
 [![Dependency Status](https://gemnasium.com/abonas/kubeclient.svg)](https://gemnasium.com/abonas/kubeclient)
 
 A Ruby client for Kubernetes REST api.
-The client supports GET, POST, PUT, DELETE on nodes, pods, secrets, services, replication controllers, namespaces, resource quotas, limit ranges, endpoints, persistent volumes, persistent volume claims, component statuses and service accounts.
+The client supports GET, POST, PUT, DELETE on all the entities available in kubernetes in both the core and group apis.
```

Comment:
+1 to move in the long term to the Kubernetes/OpenShift nomenclature, of course it's going to break the API (unless we'll support both) so we should consider it for a major version change.